regolith/tools.py

```diff
@@ -1567,3 +1567,25 @@ def grant_burn(grant, appts, grant_begin, grant_end, begin_date=None, end_date=N
                    "ss_days": round(ss_val, 2)}
             grant_amounts.append(gvals)
     return grant_amounts
+
+def validate_meeting(meeting, date):
+    """
+    Validates a meeting by checking is it has a journal club doi, a presentation link, and a presentation
+    title. This function will return nothing is the meeting is valid, otherwise it will raise a ValueError. 
+
+    Parameters
+    ----------
+    meeting: dict
+        The meeting object that needs to be validated
+    date: datetime object
+        The date we want to use to see if a meeting has happened or not
+
+    """
+    meeting_date = date_parser.parse(meeting.get('_id')[3:]).date()
```

Comment:
The meetings do not have any date attributes. The only mention of the meeting's date is in its `_id` which does something like `grpYYYY-MM-DD` so that is how I get the date.

Reply:
no year, month day?

Reply:
> 
> 
> > looks good. One comment. Also, please could we have that smoke test on the test_helpers just to make Codecov happy? Thanks.
> 
> What do you mean by a smoke test? Should I create some type of fake flag that immediately ends the `sout` function before the other function that raises the error gets called?



> 
> 
> > looks good. One comment. Also, please could we have that smoke test on the test_helpers just to make Codecov happy? Thanks.
> 
> What do you mean by a smoke test? Should I create some type of fake flag that immediately ends the `sout` function before the other function that raises the error gets called?

Sorry, not being clear there.  We need a test in ``test_helpers`` in ``test_helpers.py`` so codecov thinks all the lines of code are being tested, but it can be a lightweight test....a "smoke test" is just a very minimal test that doesn't check all the edge-cases etc.